content/work/estimating-engine.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Estimating Engine
+sub: Create & manage custom manufacturing quotes
+logo:
+image:
+  src: work/estimating-engine/estimating-engine.jpg
+  alt: manufacturing quote details
+  position: top
+client: &client General Stamping & Metalworks
+date: 2020-01-20
+end: 2021-12-01
+tags:
+  - Manufacturing Sector
+people:
+  - &ryan
+    name: Ryan Hochstetler
+    face: james-weber.jpg
+    url: https://www.linkedin.com/in/ryanhochstetler/
+    title: Information Systems Administrator
+    venue: *client
+press:
+  - text: |
+      @@@
+    <<: *ryan
+    slug: configurations
```

Comment:
uhoh, can we get an actual quote, title, and photo here?

Reply:
@mirisuzanne Yep. I'm aware Top Dog and Estimating Engine don't have the quote info they need. Left a note above about that. Looks like a only tagged Stacy. That's the oops. 

>>"We could hide Top Dog and Estimating Engine until we have quotes or we could remove the quotes section for now from those two projects. We would have to deal with a work list layout issue if we want to publish those two without quotes. Thoughts?"

Reply:
I'd say we hide them for now, so we can merge this -- and then open a new issue for how to handle featured clients without quotes?